src/librustdoc/html/render/context.rs

```diff
@@ -167,13 +167,13 @@ impl<'tcx> Context<'tcx> {
         "../".repeat(self.current.len())
     }
 
-    fn render_item(&self, it: &clean::Item, pushname: bool) -> String {
+    fn render_item(&self, it: &clean::Item, is_module: bool) -> String {
```

Comment:
This is wrong, you need to also negate the boolean that gets passed in. We want to only push the name of the item if this is *not* a module (otherwise if it were a module, we'd show it twice).